minecraft_servers/craftergang/manifest.json

```diff
@@ -1,102 +1,108 @@
 {
-  "server_name": "craftergang",
-  "nice_name": "Craftergang",
-  "direct_ip": "craftergang.de",
-  "server_wildcards": [
-    "craftergang.net",
-    "citybuild.craftergang.de"
-  ],
-  "supported_languages": [
-    "de"
-  ],
-  "social": {
-    "discord": "https://discord.gg/ThCt8Pq",
-    "teamspeak": "craftergang.de",
-    "tiktok": "craftergang",
-    "youtube": "https://www.youtube.com/@craftergang",
-    "instagram": "craftergang"
-  },
-  "gamemodes": {
-    "cbdesert": {
-      "name": "CB Desert",
-      "color": "#0d6914",
-      "command": "/gametp cb1",
-      "versions": "1.8<1.20.4"
+    "server_name": "craftergang",
+    "nice_name": "Craftergang",
+    "direct_ip": "craftergang.de",
+    "server_wildcards": [
+        "craftergang.net",
+        "citybuild.craftergang.de"
+    ],

```

Comment:
The server wildcards have already been approved by DoJapHD and are already added by previous posts. (See in LabyMod:master -> Craftergang -> manifest.json)

Reply:
Then it was probably overlooked.
In the documentation of the manifest you can see the scheme according to which wildcards must be added here (**%.**your.tld): https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md

WIldcard means that users can join on **ANY** subdomain of a main domain. The `%` therefore stands for every conceivable subdomain. However, if joining is only possible via individual subdomains, this is **NOT** a wildcard. See https://en.wikipedia.org/wiki/Wildcard_DNS_record for more reference.

Reply:
i have changed the "citybuild.craftergang.de" to "%.craftergang.de" is that okay. A question more, is "craftergang.net" okay?

Reply:
The craftergang.net domain doesn't have any DNS record. So absolute not. But in general, I'm not quite sure right now. In any case, it would be best to make a real wildcard domain out of it.

But `%.craftergang.de` doesn't seem to be a real wildcard:
![image](https://github.com/LabyMod/server-media/assets/37186532/c6a897b2-c743-47b9-afd5-c36512fa40e7)

Reply:
But the `craftergang.net` domain hasn't even an A record on the the base domain:
![image](https://github.com/LabyMod/server-media/assets/37186532/bc10da31-0ca4-496a-9c39-08dad0ca1cc2)

Reply:
![image](https://github.com/LabyMod/server-media/assets/81524111/806f3b8e-a643-4ac1-9937-b99bcf998d50)
![image](https://github.com/LabyMod/server-media/assets/81524111/f41b6108-f3ea-4ba8-ab31-3411ec74b7df)

It's the .net Domain and this appears to be displayed correctly. So it actually works perfectly.

Reply:
![image](https://github.com/LabyMod/server-media/assets/81524111/4b776693-c824-4eae-85af-81e613040346)

Reply:
**But it has entries. I guess they were not displayed because of the DDOS protection.**

Reply:
Ah okay, I forgot to check against SRV records - sry my fault

Reply:
Can you merge? (:

Reply:
:)